src/contributing/release_process.md

```diff
@@ -127,6 +127,47 @@ perform whatever testing is necessary to verify that the new feature has reached
 quality level for the release. Systems integrators and known adopters of Parsec will also be
 notified that a new release is available for integration testing.
 
+As part of the release testing, we run the end-to-end tests on a RaspberryPi for the following
+backends:
+
+- mbed crypto
+- TPM
+- Nitrokey HSM
+
+Further platforms and backends will be added to this in future releases. The results of the release
+testing are critical in determining if the parsec service and the tool is ready for release, and
+whether it meets the requirements and expectations of the end users.
+
+The following failures are expected in the RaspberryPi platform for the specific backends.
```

Comment:
Do we understand the failures? Are they actual Parsec defects or specific interop issues with the NitroKey HSM? If the failures are deemed acceptable, do we have a way to subset the e2e tests such that they are excluded from the release checking? I see it as a problem that we have a cascade of "known" failures, because it puts the onus on the engineer to manually inspect for regressions. I appreciate that this is not so much a code review comment, but rather an observation about the process itself. For the time being, if we have to document it this way, I'm okay with it in terms of this PR.

Reply:
Created #676 and #677 for the detailed investigation before we add this to the parsec book.